segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/StorageTestBase.java

```diff
@@ -40,17 +47,18 @@
 import static io.pravega.test.common.AssertExtensions.assertThrows;
 
 import static io.pravega.shared.NameUtils.INTERNAL_NAME_PREFIX;
+
 /**
  * Base class for testing any implementation of the Storage interface.
  */
 public abstract class StorageTestBase extends ThreadPooledTestSuite {
     //region General Test arguments
-
     protected static final Duration TIMEOUT = Duration.ofSeconds(30);
     protected static final long DEFAULT_EPOCH = 1;
     protected static final int APPENDS_PER_SEGMENT = 10;
     protected static final String APPEND_FORMAT = "Segment_%s_Append_%d";
     private static final int SEGMENT_COUNT = 4;
+    private static final Random RANDOM = new Random();
```

Comment:
In your tests, always initialize your RNG with a constant. That way, if you get a failure, it will be easy to reproduce since you'll always have the same input.